source/B2BApi.AppTests/Fixtures/B2BApiAppFixture.cs

```diff
@@ -0,0 +1,264 @@
+                .Add($"{IntegrationEventsOptions.SectionName}__{nameof(IntegrationEventsOptions.TopicName)}", topic.Name))
+            .AddSubscription("subscription")
+            .Do(subscription => appHostSettings.ProcessEnvironmentVariables
+                .Add($"{IntegrationEventsOptions.SectionName}__{nameof(IntegrationEventsOptions.SubscriptionName)}", subscription.SubscriptionName))
+            .CreateAsync();
+        await ServiceBusResourceProvider
+            .BuildQueue("edi-inbox")
+            .Do(queue => appHostSettings.ProcessEnvironmentVariables
+                .Add($"{EdiInboxOptions.SectionName}__{nameof(EdiInboxOptions.QueueName)}", queue.Name))
+            .CreateAsync();
+        var wholesaleInboxQueueResource = await ServiceBusResourceProvider
+            .BuildQueue("wholesale-inbox")
+            .Do(queue => appHostSettings.ProcessEnvironmentVariables
+                .Add($"{WholesaleInboxOptions.SectionName}__{nameof(WholesaleInboxOptions.QueueName)}", queue.Name))
+            .CreateAsync();
+        await ServiceBusResourceProvider
+            .BuildQueue("incomming-messages")
+            .Do(queue => appHostSettings.ProcessEnvironmentVariables
+                .Add($"{IncomingMessagesQueueOptions.SectionName}__{nameof(IncomingMessagesQueueOptions.QueueName)}", queue.Name))
+            .CreateAsync();
+
+        // => Receive messages on Wholesale Inbox Queue
+        await ServiceBusListenerMock.AddQueueListenerAsync(wholesaleInboxQueueResource.Name);
+
+        // Create and start host
+        AppHostManager = new FunctionAppHostManager(appHostSettings, TestLogger);
+        StartHost(AppHostManager);
+
+        // Create durable client when TaskHub has been created
+        DurableClient = DurableTaskManager.CreateClient(taskHubName: TaskHubName);
+    }
+
+    public async Task DisposeAsync()
+    {
+        AppHostManager.Dispose();
+        DurableTaskManager.Dispose();
+        AzuriteManager.Dispose();
+        await ServiceBusResourceProvider.DisposeAsync();
+        await DatabaseManager.DeleteDatabaseAsync();
+    }
+
+    /// <summary>
+    /// Use this method to attach <paramref name="testOutputHelper"/> to the host logging pipeline.
+    /// While attached, any entries written to host log pipeline will also be logged to xUnit test output.
+    /// It is important that it is only attached while a test i active. Hence, it should be attached in
+    /// the test class constructor; and detached in the test class Dispose method (using 'null').
+    /// </summary>
+    /// <param name="testOutputHelper">If a xUnit test is active, this should be the instance of xUnit's <see cref="ITestOutputHelper"/>;
+    /// otherwise it should be 'null'.</param>
+    public void SetTestOutputHelper(ITestOutputHelper testOutputHelper)
+    {
+        TestLogger.TestOutputHelper = testOutputHelper;
+    }
+
+    public void EnsureAppHostUsesFeatureFlagValue(bool featureFlagValue)
```

Comment:
```suggestion
    public void EnsureAppHostUsesFeatureFlagValue(bool enableCalculationCompletedEvent)
```
"featureFlagValue" is very generic, maybe "enableCalculationCompletedEvent" since that is the feature we unsure is working?

Reply:
Fixed